src/rgw/rgw_common.cc

```diff
@@ -25,6 +29,13 @@
 
 #define dout_subsys ceph_subsys_rgw
 
+#define POLICY_ACTION           0x01
+#define POLICY_RESOURCE         0x02
+#define POLICY_ARN              0x04
+#define POLICY_STRING           0x08
+
+using namespace boost;
+
```

Comment:
Boost is rather large. Since we're tracking current, it would probably be better to import just the names we need and not have potential clashes down the road.